plugin/federation/federation.go

```diff
@@ -148,6 +158,15 @@ func (f *federation) MutateSchema(s *ast.Schema) error {
 		Type: ast.NonNullNamedType("_Service", nil),
 	}
 	s.Query.Fields = append(s.Query.Fields, _serviceDef)
+}
+
+// MutateSchema creates types and query declarations
+// that are required by the federation spec.
+func (f *federation) MutateSchema(s *ast.Schema) error {
+	if len(f.Entities) > 0 {
```

Comment:
Minor: could you add a comment here and in InjectSources about why there might not be any entities?

Reply:
Sure thing!